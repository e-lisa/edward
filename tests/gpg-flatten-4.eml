From: No One <noone@localhost>
To: No One <noone@localhost>
Subject: =?UTF-8?B?44GT44KT44Gr44Gh44Gv44CB44GK5YWD5rCX44Gn44GZ44GL77yf?=

DO YOU READ ME CAPTAIN??!!1 o_O

-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

hi, i didn't encrypt this part. is that a mistake?

bye!

-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1

iJwEAQECAAYFAlWthWEACgkQsT5tL2pPbuhyWgQApzhIt4EM4FrqCibNI7ZS1V73
yhml95bw5PXq2fB7wU2I4pkM30T2kpMxdtfHdPXHQSzvuo27IhrUOH7CUtvj0f23
CllwAGoUmwmdXOLeVvgjwyPzSVAdWLO8/231fS1M8J42v8GV6w78hmq+/Cck2oQE
p4/91AVSikY7bwMrGpU=
=Tavv
-----END PGP SIGNATURE-----

-----BEGIN PGP MESSAGE-----
Version: GnuPG v1

hIwDWouAYlp2iv8BBADAXsI9Y/miNCVnseKDNZVil3VK3SUbJ4DYoVDVcymcStUw
+bmk3XSk5hqap3CfSKdSvdnxSmaKzYOdpv03kiRtK2gjnakcKxk4pGz85UM5t8ZZ
3estj/Ca40kvSHZAYelgC1W8JshDOyOrS93acfUgJxKQMdxhGRjVAW1YbQWFyNLA
3QEJJeg6obNktEbhXCSlupZFAKJQgdAPDOWYgNeFnTkS4MttM+G0ABx/o8JjeIl5
wYDucFBDHXSuld9RBzQ1QLOUhB8BOkCq6XFgwaaTKTBT7SQ19n0G8RpZmJ/BX5Ok
msi9nlRm9z6Irjtx9bDudzUfpCzO/YeYBN1J/1o3pw8f/otzOMfRHBStkdo0V9i5
9JS77QRkSnM4hVrZuvfplYa5iRftdYehLMaMk7k9gOY+B5WEMdgV9ZWRrWOXmqTK
NyB1fIWgKNEqCgZnBFPS6Es6kUXUNnFvsvFMPg5WXx+HBJ3YqhNzsdVegagxjE9F
wi7qHlcHbRGFv7Zz+BsiPQ3M3tQgwBAd1c/d/6CIdZDArvsTZiSdkszpfeziVj2p
ZKhWUBUIpqP59MMlftD19EFTtJm8mpc/gBH074nEwzXdLFKg08Vb5fH2zMRo37w4
HZzUHFJaj8VMIjQPRXqD6ELclHJbO7ASTyh8S+v9Iv+t4PpJXn8odwAm08bd//e9
Ch/H9RAlS/22W4KlpZdIvwz7JmwVE9T4Xq1e22Df
=Sn5u
-----END PGP MESSAGE-----

