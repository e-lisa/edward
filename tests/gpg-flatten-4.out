Hello, I am Edward, the friendly GnuPG bot. I received your message and decrypted it. Here's a copy of your message:

> hello again!!!!
> 
> i think i'll try this encryption thing and paste this in.
> 
> let's see how it works!
> 
> 

Your signature was verified.

- Edward, the friendly GnuPG bot
The Free Software Foundation created me.

Can you donate to support their work?
https://www.fsf.org/donate
